### ISSUE TITLE
[tune] Provide an example of using Dask on Ray with Tune

### ISSUE THREAD

**Issue by richardliaw** [ray-project/ray#11564]
### Describe your feature request

In Ludwig, different hyperparam configs can result in different preprocessing, so we may want to run Dask on Ray within a Ray Tune job.  

We should provide a simple example (training xgboost on dask-on-ray) in Tune so that this is easy to do in the future.

cc @tgaddair

**Comment by rkooo567** [comment]
cc @clarkzinzow Maybe you can take this over next week?

**Comment by clarkzinzow** [comment]
I'd be happy to! 😁

**Comment by clarkzinzow** [comment]
Hey @krfricke, sorry that this was lost! Given that this will probably involve more XGBoost and Tune code/knowledge than Dask-on-Ray, would y'all be able to take this on?

**Comment by dynamicwebpaige** [comment]
It looks like this issue might have been solved by this [XGBoost + Dask-on-Ray + Tune example](https://github.com/ray-project/ray/blob/3cb85859cd4eb23a820871c3b00ba8215d6d59a1/doc/source/ray-core/examples/dask_xgboost/dask_xgboost.ipynb). Would it make sense to refer to this example in the ray.io documentation (if it isn't mentioned explicitly already)?

**Comment by cszhu** [comment]
Hello! This P2 issue has seen no activity in at least 2 years. It will be closed in 2 weeks as part of ongoing cleanup efforts.

Please remove the `pending-cleanup` label if you believe this issue should remain open.

Thanks for contributing to Ray!
